project/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -64,6 +89,13 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.getWriter().println(dataPackageJson);
   }
 
+  /**
+   * Resets the flag for whether the user input address is relevant but nonspecific.
+   */
+  private void resetRelevantNonspecificFlag() {
+    this.isAddressRelevantButNonspecificOrNonresidential = false;
```

Comment:
nit. rename this to isAddressRelevantButNotSpecificOrResidential

Reply:
Thanks!